ads/pubmine.js

```diff
@@ -22,8 +22,10 @@ const pubmineOptional = ['section', 'pt', 'ht'],
 
 /**
  * @param {!Object} data
+ * @param {!Window} global
+ * @param {!Function} [scriptLoader=loadScript]
  */
-function initMasterFrame(data) {
+function initMasterFrame(data, global, scriptLoader = loadScript) {
```

Comment:
I understand why global may not behave how we would expect here, but I am curious why `loadScript` needs to be passed around. Are you just being extra cautious here?

Reply:
It's an easy way to pass a spy. I check if `loadScript` was called in unit tests.

Reply:
Got it. Thanks for clarifying.

Reply:
agree, `scriptLoader ` is not needed here

Reply:
Let's decide what we should do here. In my opinion, it's fine to pass the function. If stubbing `loadScript` in tests is more preferable I can implement it.

Reply:
Did you see the comment here https://github.com/ampproject/amphtml/pull/22431#issuecomment-494877854